src/test/java/org/takes/rq/ChunkedInputStreamTest.java

```diff
@@ -133,7 +133,7 @@ void ignoresParameterAfterSemiColon() throws IOException {
                     data,
                     ChunkedInputStreamTest.END_OF_CHUNK,
                     ""
-                ).asString(),
+                ).toString(),
```

Comment:
@andreoss Is it mandatory to use `toString` ?

Reply:
@baudoliver7 The signature has changed `asString` now throws `Exception` from `IOException`. (see https://github.com/yegor256/cactoos/blob/0.35/src/main/java/org/cactoos/text/TextEnvelope.java#L62).

Reply:
@andreoss I was asking this because sometimes, `toString()` is called implicitly. But in these cases, you are right.